mpl/topology_communicator.hpp

```diff
@@ -271,8 +271,8 @@ namespace mpl::impl {
       /// \note This is a collective operation and must be called (possibly by utilizing anther
       /// overload) by all processes in the communicator.
       template<typename T>
-      irequest ineighbor_alltoall(const T *senddata, const layout<T> &sendl, T *recvdata,
-                                  const layout<T> &recvl) const {
+      irequest ineighbor_alltoall(const T *senddata, [[maybe_unused]] const layout<T> &sendl, T *recvdata,
+                                  [[maybe_unused]] const layout<T> &recvl) const {
```

Comment:
I think, not using `sendl` and `recvl` is a bug that needs to be fixed. Note that the function body is identical to the overloaded method a few lines above.

Reply:
OK, I reverted to original